### DISCUSSION TITLE
What are you all using Hyprspace for?

### DISCUSSION THREAD

**Discussion by alecbcs** [alecbcs/hyprspace#17] [Show and tell]
Hey all!

If you're using Hyprspace, what for? I've been astonished by so many people finding the project and thought this would be a fun place to share our use cases. 

So I'll go first! I'm using Hyprspace to connect a couple of geographically separated backup servers so they can find each other without having to setup any manual port forwarding rules. I'm also using a [Hyprspace Relay](https://github.com/hyprspace/relay) to create a public endpoint to a server so that I can share files with friends over a Nextcloud instance. This has worked really well! For the price of a small cloud VM I've been able to make a Nextcloud instance with much more hard drive space than I could afford on a cloud instance.

Best,
Alec

**Comment by Winterhuman** [comment]
I'm planning to use Hyprspace to connect my laptop to my desktop at home to access a Nextcloud server (Nextcloud's not even setup yet, still learning), was going to use ZeroTier instead until I found this project; I like to minimize the number of accounts I create, never realised that Hyprspace Relay existed until now! Perhaps add a mention of it in the Readme?

**Reply by alecbcs** [reply]
Very cool! I'd never heard of ZeroTier before so thanks for the introduction! And good luck with building a Nextcloud instance! If you're looking for any advice on that front I'd recommend looking into Nextcloud's docker-compose examples as a great starting point for getting all of the components running in one step. 

I've kinda been waiting to more extensively test [Relay](https://github.com/hyprspace/relay) before I announce it to the world and stick it within the Hyprspace README but you're definitely right that once it's a little more mature. We should talk about it more within this Repository!
